File: 657.py
class Solution:
    def judgeCircle(self, moves: str) -> bool:
        x = 0
        y = 0
        for letter in moves:
            if letter == "R":
                x += 1
            elif letter == "L":
                x -= 1
            elif letter == "U":
                y += 1
            elif letter == "D":
                y -= 1
        if x == 0 and y == 0:
            return True
        else:
            return False

File: easy/657.java
class Solution {
    public boolean judgeCircle(String moves) {
        int x = 0;
        int y = 0;
        for (int i=0;i<moves.length();i++) {
            char move = moves.charAt(i);
            switch (move) {
                case 'U':
                    x += 1;
                    break;
                case 'D':
                    x -= 1;
                    break;
                case 'L':
                    y -= 1;
                    break;
                case 'R':
                    y += 1;
                    break;
            }
        }
        if (x ==0 && y==0) {
            return true;
        } else {
            return false;
        }
     }
}
